src/main/java/weka/classifiers/trees/j48It/C45PruneableClassifierTreeIt.java

```diff
@@ -28,10 +28,10 @@ public class C45PruneableClassifierTreeIt extends
 	/** Build the tree level by level up to a maximum of depth levels. 
 	 * Set m_levelByLevel_growth to 0 to use the optimal number of levels.
```

Comment:
Portzierto, hau ez dut ulertu... Set 0 to use the optimal number???? Baina nola dakigu zein den maila kopururik optimoa? Jakingo bagenu...
Zer esan nahi duzu horrekin?